addons/account_followup/account_followup.py

```diff
@@ -384,120 +349,106 @@ def _get_followup_overdue_query(self, cr, uid, args, overdue_only=False, context
+                                     "Can be practical to set manually e.g. to see if he keeps "
+                                     "his promises.")
+    unreconciled_aml_ids = fields.One2many('account.move.line', 'partner_id', domain=['&', ('reconciled', '=', False), '&',
+                        ('account_id.deprecated','=', False), '&', ('account_id.internal_type', '=', 'receivable')])
+    latest_followup_date = fields.Date(compute='_get_latest',
+                                       help="Latest date that the follow-up level of the partner was changed",
+                                       store=False, multi="latest")
+    latest_followup_level_id = fields.Many2one('account_followup.followup.line', string="Latest Follow-up Level",
+        help="The maximum follow-up level", store=True, multi="latest")
+    latest_followup_level_id_without_lit = fields.Many2one('account_followup.followup.line', string="Latest Follow-up Level without litigation",
+        help="The maximum follow-up level without taking into account the account move lines with litigation",
+        store=True, multi="latest")
+    payment_amount_due = fields.Float(compute='_get_amounts_and_date',
+                                      string="Amount Due",
+                                      store=False, multi="followup",
+                                      search='_payment_due_search')
+    payment_amount_overdue = fields.Float(compute='_get_amounts_and_date',
+                                          string="Amount Overdue",
+                                          store=False, multi="followup",
+                                          search='_payment_overdue_search')
+    payment_earliest_due_date = fields.Date(compute='_get_amounts_and_date',
+                                            string="Worst Due Date",
+                                            multi="followup",
+                                            search='_payment_earliest_date_search')
+
+
+class AccountConfigSettings(models.TransientModel):
     _name = 'account.config.settings'
     _inherit = 'account.config.settings'
-    
-    def open_followup_level_form(self, cr, uid, ids, context=None):
-        res_ids = self.pool.get('account_followup.followup').search(cr, uid, [], context=context)
-        
-        return {
-                 'type': 'ir.actions.act_window',
-                 'name': 'Payment Follow-ups',
-                 'res_model': 'account_followup.followup',
-                 'res_id': res_ids and res_ids[0] or False,
-                 'view_mode': 'form,tree',
-         }
+
+    def open_followup_level_form(self):
+        res_ids = self.env['account_followup.followup'].search([])
+        return {'type': 'ir.actions.act_window',
+                'name': 'Payment Follow-ups',
+                'res_model': 'account_followup.followup',
+                'res_id': res_ids and res_ids[0] or False,
```

Comment:
Searches can often be limited by using limit=1.
Here 'res_ids' contains recordset(s) not ids.